client/src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { useState } from "react";
+import { MenuItems } from "./MenuItems";
+import "./Navbar.css";
+
+function Navbar() {
+  {
+    /* This is the hook to change the state of a menu item */
+  }
+  const [clicked, setState1] = useState(false);
```

Comment:
This is was code I copied from the tutorial but he implemented a button along with it. I didn't do the button part so that's why it's unused right now.